pkg/kubectl/cmd/exec.go

```diff
@@ -174,6 +174,12 @@ func (p *ExecOptions) Run() error {
 		return err
 	}
 
+	started := pod.Status.StartTime != nil
```

Comment:
When a pod terminates I guess the kubelet will return `pod not found` again although `nodeName` is still set. I guess we have to put logic on both sides to cover all cases, especially for the one in https://github.com/openshift/origin/issues/8472#event-681794952. I am going to remove the check for `started` and fall back to the kubelet error value for that.